src/core/constants.ts

```diff
@@ -49,6 +49,44 @@ export const SWA_AUTH_CONTEXT_COOKIE = `StaticWebAppsAuthContextCookie`;
 export const SWA_AUTH_COOKIE = `StaticWebAppsAuthCookie`;
 export const ALLOWED_HTTP_METHODS_FOR_STATIC_CONTENT = ["GET", "HEAD", "OPTIONS"];
 
+// Custom Auth constants
+export const SUPPORTED_CUSTOM_AUTH_PROVIDERS = ["google", "github", "aad"];
+// Full name is required in staticwebapp.config.json's schema so we will normalize it to aad
+export const AAD_FULL_NAME = "azureActiveDirectory";
```

Comment:
Change to entraId or similar - AAD is the old name - need new branding.

Reply:
I can only change the variable name to `ENTRAID_FULL_NAME` for now. The configuration json is still expecting `azureActiveDirectory` so until that is changed, I cannot re-brand the expected string.